src/apb/cli.py

```diff
@@ -42,14 +42,20 @@ def subcmd_init_parser(parser, subcmd):
     )
 
     subcmd.add_argument(
-        '--bindable', action='store_true', dest='bindable',
-        help=u'Make application bindable', default=False
+        '--not-bindable', action='store_false', dest='bindable',
+        help=u'Make application not bindable on the spec.', default=True
```

Comment:
I think I missed something. Does this mean `--bindable` is no longer valid? or does this mean *both* `--bindable` and `--not-bindable` are valid?